NitroxServer-Subnautica/Serialization/ResourceAssets/MonoBehaviourParsers/ReefbackSlotsDataParser.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxServer_Subnautica.Serialization.ResourceAssets.MonoBehaviourParsers.Abstract;
+using AssetsTools.NET;
+using AssetsTools.NET.Extra;
+using NitroxServer_Subnautica.GameLogic.Entities.Spawning.EntityBootstrappers;
+using NitroxServer_Subnautica.Serialization.ResourceAssets;
+using NitroxServer.UnityStubs;
+
+namespace NitroxServer_Subnautica.Serialization.ResourceAssets.MonoBehaviourParsers
+{
+    public class ReefbackSlotsDataParser : MonoBehaviourParser
+    {
+        public override void Parse(AssetsFileInstance instance, long gameObjectPathId, ResourceAssets resourceAssets)
```

Comment:
It looks like most of the code parses the same data structure as `DefaultMonoBehaviourParser`. Would it be possible to merge it (together with the "reefbackCreature" loop below) and decrease duplication as well as future code maintenance?

Even if there are slight differences which I have not noticed, splitting it out into a few common functions goes a long way already.